src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_TripSearchScreen/components/Results.tsx

```diff
@@ -73,7 +73,7 @@ const Results: React.FC<Props> = ({
 
   if (errorType) {
     return (
-      <View style={styles.container}>
```

Comment:
Looks good however `styles.container` was only used here, so it might not be in use by other components.